packages/backend/server/src/core/user/resolver.ts

```diff
@@ -137,3 +140,111 @@ export class UserResolver {
     return { success: true };
   }
 }
+
+@InputType()
+class ListUserInput {
+  @Field(() => Int, { nullable: true, defaultValue: 0 })
+  skip!: number;
+
+  @Field(() => Int, { nullable: true, defaultValue: 20 })
+  first!: number;
```

Comment:
unusal naming? I recommend using `limit`

Reply:
we didn't follow the best practice of pagination as graphql spec recommends, anyway